### ISSUE TITLE
Расширенный редактор описания события

### ISSUE THREAD

**Issue by artech53** [artsiom-burkun/DropCalendar#2]
Не смог найти где добавить самому. потому пишу тут:)
Есть необходимость в одной из двух функций:
1. Полноценный WYSIWYG редактор для подробного описания события
2. Простой интерфейс выбора странички с описанием события. Т.е. я создаю страничку, и по ID прикрепляю её к событию. а потом уже в чанке события как-то обрабатываю данные из странички. Можно и руками ID вбивать - но это, всё же, несколько костыль.

Просто не могу найти. где окно редактирования события из панели администратора, редактируется:(

**Comment by artsiom-burkun** [comment]
Я так понимаю, что вам нужен WYSIWYG редактор или доступ к стандартному редактору из панели администратора. Думаю, что это можно сделать. 
По срокам и времени. Новая версия компонента DropCalendar будет основана на 4-й версии fullcalendar.io. Обновление компонента ожидается после нового года.

**Comment by artech53** [comment]
Да, мне бы хотелось добавлять описание события в виде форматированного текста.Сейчас это возможно с применением HTML-тегов, но это не удобно.Возможность создания форматированного описания события будет очень кстати.  05.11.2019, 13:40, "artsiom-burkun" <notifications@github.com>:Я так понимаю, что вам нужен WYSIWYG редактор или доступ к стандартному редактору из панели администратора. Думаю, что это можно сделать.По срокам и времени. Новая версия компонента DropCalendar будет основана на 4-й версии fullcalendar.io. Обновление компонента ожидается после нового года.—You are receiving this because you authored the thread.Reply to this email directly, view it on GitHub, or unsubscribe.  -- Максим Юдин+7-960-205-18-86yudin.mn@mail.ru Великий Новгород
